test/integration/fuzz/lockup-tranched/createWithDurations.t.sol

```diff
@@ -0,0 +1,146 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.22 <0.9.0;
+
+import { Lockup, LockupTranched } from "src/types/DataTypes.sol";
+
+import { CreateWithDurations_Integration_Shared_Test } from "../../shared/lockup-tranched/createWithDurations.t.sol";
+import { LockupTranched_Integration_Fuzz_Test } from "./LockupTranched.t.sol";
+
+contract CreateWithDurations_LockupTranched_Integration_Fuzz_Test is
+    LockupTranched_Integration_Fuzz_Test,
+    CreateWithDurations_Integration_Shared_Test
+{
+    function setUp()
+        public
+        virtual
+        override(LockupTranched_Integration_Fuzz_Test, CreateWithDurations_Integration_Shared_Test)
+    {
+        LockupTranched_Integration_Fuzz_Test.setUp();
+        CreateWithDurations_Integration_Shared_Test.setUp();
+    }
+
+    struct Vars {
+        uint256 actualNextStreamId;
+        address actualNFTOwner;
+        uint256 actualProtocolRevenues;
+        Lockup.Status actualStatus;
+        Lockup.CreateAmounts createAmounts;
+        uint256 expectedNextStreamId;
+        address expectedNFTOwner;
+        uint256 expectedProtocolRevenues;
+        Lockup.Status expectedStatus;
+        address funder;
+        uint128 initialProtocolRevenues;
+        bool isCancelable;
+        bool isSettled;
+        LockupTranched.Tranche[] tranchesWithTimestamps;
+        uint128 totalAmount;
+    }
```

Comment:
Would it be overkill to reuse these `Vars` declarations across `Lockup` test files?

Cc @andreivladbrg, @smol-ninja

Reply:
I think yes. Each test with its own struct is easier to maintain compared to a large struct. As we keep updating tests, it could become time-intensive to identify and remove unused variables whereas in the current form, it is easier to keep it lean.

Reply:
Yeah, you're right. Also, it's handy to see the test variables declared right above the test.